Corona-Warn-App/src/main/res/layout/home_statistics_cards_incidence_layout.xml

```diff
@@ -10,15 +10,83 @@
 
         <TextView
             android:id="@+id/title"
-            style="@style/headline5"
+            style="@style/StatisticsCardPrimaryTitle"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginStart="24dp"
-            android:layout_marginTop="24dp"
+            android:layout_marginStart="@dimen/card_padding"
+            android:layout_marginTop="@dimen/spacing_medium"
             android:focusable="false"
-            android:text="Incidence"
+            android:text="@string/statistics_card_incidence_title"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/icon_info"/>
+
+        <androidx.appcompat.widget.AppCompatImageView
```

Comment:
Not sure whether we need to use `AppCompatImageView`, `</ImageView> ` may be sufficient